Tweeter.Tests/DAL/TweeterRepoTests.cs

```diff
@@ -62,6 +66,33 @@ public void RepoEnsureCanCreateInstance()
         [TestMethod]
         public void RepoEnsureICanGetUsernames()
         {
+        
+            var user1 = new ApplicationUser { UserName = "James123", Email = "james123@email.com" };
+            users.Add(user1);
+
+
+            TweeterRepository repo = new TweeterRepository(mock_user_manager_context.Object);
+            List<string> all_users = repo.GetAllAppUsers();
+            
+            
+
+            int expected_count = 1;
+            int actual_count = all_users.Count();
+
+            Assert.AreEqual(expected_count, actual_count);
+            Assert.IsInstanceOfType(all_users, typeof(List<string>));
+        }
+
+        [TestMethod]
+        public void RepoEnsureCanVerifyUsernameExists()
```

Comment:
if you want to be thorough, you should make a test where the name *doesn't* exist and you should Assert that it's false.